2024-06-22 02:11:36,153 INFO    StreamThr :3536 [internal.py:wandb_internal():87] W&B internal server running at pid: 3536, started at: 2024-06-22 02:11:36.153272
2024-06-22 02:11:36,155 DEBUG   HandlerThread:3536 [handler.py:handle_request():144] handle_request: status
2024-06-22 02:11:36,157 INFO    WriterThread:3536 [datastore.py:open_for_write():85] open: /home/animated/Projects/Sailbot/src/object_detection/object_detection/wandb/offline-run-20240622_021136-en4psasg/run-en4psasg.wandb
2024-06-22 02:11:36,161 DEBUG   HandlerThread:3536 [handler.py:handle_request():144] handle_request: run_start
2024-06-22 02:11:36,179 DEBUG   HandlerThread:3536 [system_info.py:__init__():31] System info init
2024-06-22 02:11:36,179 DEBUG   HandlerThread:3536 [system_info.py:__init__():46] System info init done
2024-06-22 02:11:36,179 INFO    HandlerThread:3536 [system_monitor.py:start():183] Starting system monitor
2024-06-22 02:11:36,179 INFO    SystemMonitor:3536 [system_monitor.py:_start():147] Starting system asset monitoring threads
2024-06-22 02:11:36,180 INFO    HandlerThread:3536 [system_monitor.py:probe():204] Collecting system info
2024-06-22 02:11:36,180 INFO    SystemMonitor:3536 [interfaces.py:start():187] Started cpu monitoring
2024-06-22 02:11:36,181 INFO    SystemMonitor:3536 [interfaces.py:start():187] Started disk monitoring
2024-06-22 02:11:36,182 INFO    SystemMonitor:3536 [interfaces.py:start():187] Started gpu monitoring
2024-06-22 02:11:36,183 INFO    SystemMonitor:3536 [interfaces.py:start():187] Started memory monitoring
2024-06-22 02:11:36,184 INFO    SystemMonitor:3536 [interfaces.py:start():187] Started network monitoring
2024-06-22 02:11:37,156 INFO    SenderThread:3536 [sender.py:finish():1504] shutting down sender
2024-06-22 02:11:37,160 INFO    WriterThread:3536 [datastore.py:close():298] close: /home/animated/Projects/Sailbot/src/object_detection/object_detection/wandb/offline-run-20240622_021136-en4psasg/run-en4psasg.wandb
2024-06-22 02:11:37,160 ERROR   StreamThr :3536 [internal.py:wandb_internal():175] Thread HandlerThread:
Traceback (most recent call last):
  File "/home/animated/miniforge3/lib/python3.10/site-packages/wandb/sdk/internal/internal_util.py", line 49, in run
    self._run()
  File "/home/animated/miniforge3/lib/python3.10/site-packages/wandb/sdk/internal/internal_util.py", line 100, in _run
    self._process(record)
  File "/home/animated/miniforge3/lib/python3.10/site-packages/wandb/sdk/internal/internal.py", line 280, in _process
    self._hm.handle(record)
  File "/home/animated/miniforge3/lib/python3.10/site-packages/wandb/sdk/internal/handler.py", line 136, in handle
    handler(record)
  File "/home/animated/miniforge3/lib/python3.10/site-packages/wandb/sdk/internal/handler.py", line 146, in handle_request
    handler(record)
  File "/home/animated/miniforge3/lib/python3.10/site-packages/wandb/sdk/internal/handler.py", line 694, in handle_request_run_start
    self._system_monitor.probe(publish=True)
  File "/home/animated/miniforge3/lib/python3.10/site-packages/wandb/sdk/internal/system/system_monitor.py", line 207, in probe
    k: v for d in [asset.probe() for asset in self.assets] for k, v in d.items()
  File "/home/animated/miniforge3/lib/python3.10/site-packages/wandb/sdk/internal/system/system_monitor.py", line 207, in <listcomp>
    k: v for d in [asset.probe() for asset in self.assets] for k, v in d.items()
  File "/home/animated/miniforge3/lib/python3.10/site-packages/wandb/sdk/internal/system/assets/gpu.py", line 353, in probe
    info["gpu"] = pynvml.nvmlDeviceGetName(pynvml.nvmlDeviceGetHandleByIndex(0))  # type: ignore
  File "/home/animated/miniforge3/lib/python3.10/site-packages/wandb/vendor/pynvml/pynvml.py", line 1962, in wrapper
    return res.decode()
UnicodeDecodeError: 'utf-8' codec can't decode byte 0xf8 in position 0: invalid start byte
